### DISCUSSION TITLE
Snooze notifications

### DISCUSSION THREAD

**Discussion by sethfri** [community/community#3661] [Mobile]
It would be really cool to be able to snooze specific notifications until Tomorrow Morning, This Evening, or specific times, kind of like Gmail and Slack can do. It would be really helpful for managing the notification inbox in GitHub to triage some notifications for later.

Thanks!

**Comment by aconsuegra** [comment]
This is really good feedback @sethfri! 

This functionality is already available on some Android devices, like new Samsung phones. But I think it could be great if we could offer it within the app, so it would be available in all devices and platforms. I'll add it to our list of improvements.

**Reply by benjamincburns** [reply]
FWIW, I think the request doesn't just pertain to mobile, but notifications in general (e.g. when clicking the notification bell on desktop).

**Comment by ditman** [comment]
I'd love to see this feature implemented too. Some of the issues I have to deal with are not actionable until "X months" in the future, and I would like to take them off my inbox completely, until it's time to take a look at them again :)

I found https://github.com/probot/reminders that might do the trick, with the added benefit that the user I'm interacting with in the notification becomes *aware* that I snoozed it for later.

**Comment by driesvints** [comment]
Not sure why this is in the mobile section but I'd really like to see this in the web UI of GitHub notifications as well 😅

**Comment by dmurvihill** [comment]
+1, I'm getting work notifications from GitHub during my vacation 😢

**Comment by voxpelli** [comment]
I think this would solve the use case that "Save" is most often used for. Something that one needs to get back to later but with "Save" one needs to remember to review it later and can't easily separate between things that should be snoozed for weeks / months vs snoozed until later the same day / week.

Would also be kind of nice to be able to add reminders for issues and PR:s straight away, without having to get a notification first.
